swupd/rename_test.go

```diff
@@ -137,7 +137,7 @@ func TestRename(t *testing.T) {
 	for _, tc := range tests {
 		add := filelist(t, sn, tc.add)
 		remove := markdelete(filelist(t, sn, tc.remove))
-		renameDetection(&Manifest{}, add, remove, config{})
+		_ = renameDetection(&Manifest{}, add, remove, config{})
```

Comment:
You should check the error and fail the test if it is not `nil`

Reply:
Updated.